source/process/community-guidelines.rst

```diff
@@ -163,15 +163,17 @@ Help Wanted Issue Links
 
 Below is a list of permanent redirects to Help Wanted issue queries:
 
-1. `mattermost/mattermost-server` - https://mattermost.com/pl/help-wanted-mattermost-server which redirects to https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Help+Wanted%22+label%3A%22Tech%2FGo%22+is%3Aissue
+1. `mattermost` - https://mattermost.com/pl/help-wanted which redirects to https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22
 
-2. `mattermost/desktop` - https://mattermost.com/pl/help-wanted-desktop which redirects to https://github.com/mattermost/desktop/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22
+2. `mattermost/mattermost-server` - https://mattermost.com/pl/help-wanted-mattermost-server which redirects to https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Help+Wanted%22+label%3A%22Tech%2FGo%22+is%3Aissue
```

Comment:
What do you think about filtering out the ticket that are not up for grabs? This way we can make sure to not lead people on working on tickets that other contributors are already working on.

Reply:
I'm 0/5, for me the first link accomplishes that and the others give a list of all help wanted issues -- sometimes we've had contributors notice that an issue isn't active and asked if they can claim it instead (although that's happened less frequently after you started doing most of the help wanted issue reviews)

Reply:
I'm 2/5 that we should moderate the help wanted process and should be the one asking the contributor if he/she still wants to work on an issue. Given that I'm inclined to "hide" not available tickets from contributors.

Reply:
@hanzei Updated to include `Up For Grabs` label. Once approved, I'll ask marketing team to help update the redirects. Thanks!